### PR TITLE
fix: google.protobuf.Value showing unset

### PR DESCRIPTION
**Pull request by hyperyuri**
About issue #152

On server the value can be:

image

@jhump

### CODE REVIEW

File: internal/resources/webform/webform.js [hunk 1]
```diff
@@ -1923,7 +1923,7 @@ window.initGRPCForm = function(services, invokeURI, metadataURI, debug, headers)
             case "google.protobuf.BoolValue":
                 return false;
             case "google.protobuf.Value":
-                return null;
+                return "";
```
Comment (jhump) [inline internal/resources/webform/webform.js:1926]: I think `null` is a better default than empty string (as would be `{}`). Also, `null` is an acceptable value for a `google.protobuf.Value`. So perhaps the real issue is somewhere else, not properly handling the `null` value?

Comment (hyperyuri) [reply internal/resources/webform/webform.js]: yes, i will debug to see where it's not handling the null value

Comment (hyperyuri) [reply internal/resources/webform/webform.js]: we can made an if inside `addOptionalMessage` to check if `fld` is `google.protobuf.Value` to then return `addElementToForm(schema, container, parent, pathLen, value, allowMissing, fld);`

Comment (hyperyuri) [reply internal/resources/webform/webform.js]: if you agree with this solution or the other one, please tell me, than i will fix

Comment (hyperyuri) [reply internal/resources/webform/webform.js]: i already test it with the if and it works

**Review by jhump — APPROVED** [review]
LGTM

**Review by jhump — COMMENTED** [review]
Before I can merge this, you need to run `make generate`. This JS file is actually incorporated into the Go package in the generated file `internal/resources/webform/bindata.go`. And running `make generate` should update that generated file with this one line change.

**Pull request merged**